plugins/woocommerce-blocks/assets/js/blocks/product-gallery/frontend.tsx

```diff
@@ -109,8 +109,21 @@ const productGallery = {
 				attributes: true,
 			} );
 
+			const clearVariationsLink = document.querySelector(
+				'.wp-block-add-to-cart-form .reset_variations'
+			);
+
+			const handleClick = () => {
```

Comment:
Nit-pick alert: Could we perhaps change the name to something more descriptive?

How about `selectFirstImage` - I'm think this would be more in line with the way we named the `actions`: `closeDialog`, `openDialog`, `selectImage`, `selectNextImage`, `selectPreviousImage`